README.md

```diff
@@ -1,21 +1,24 @@
 # Pug Deno
 
-[Pug library](https://pugjs.org/) ported to Deno, to use in [lume](https://lumeland.github.io/)
+[Pug library](https://pugjs.org/) ported to Deno, to use in
+[lume](https://lumeland.github.io/)
```

Comment:
Maybe we can remove the "to use in lume" part to avoid misunderstandings about if this is a lume plugin or not.

Reply:
Sounds good